File: pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

/**
 * Created by a2363196581 on 2018/4/3.
 */
public interface SeckillGoodsService {
    /**
     * 返回当前正在参与秒杀的商品
     * @return
     */
    public List<TbSeckillGoods> findList();


    /**
     * 根据ID获取实体(从缓存中读取)
     */
    public TbSeckillGoods findOneFromRedis(Long id);


    /**
     * 提交订单
     * @param seckillId
     * @param userId
     */
    public void submitOrder(Long seckillId,String userId);
}

File: pinyougou-seckill-web/src/main/webapp/js/service/seckillGoodsService.js
//购物车服务层
app.service('seckillGoodsService',function($http){


    //查询秒杀所有商品
    this.findList=function(){
        return $http.post('seckillgoods/findList.do');
    }



    this.findOneFromRedis=function(id){
        return $http.post('seckillgoods/findOneFromRedis.do?id='+id);
    }



    this.submitOrder=function(id){
        return $http.post('seckillgoods/submitOrder.do?seckillId='+id);
    }
});
